src/backend/api/meals.js

```diff
@@ -1,15 +1,90 @@
-const express = require("express");
+/** @format */
+
+const express = require('express');
 const router = express.Router();
-const knex = require("../database");
+const mealsRouter = express.Router();
+const knex = require('../database');
 
-router.get("/", async (request, response) => {
+router.get('/', async (request, response) => {
   try {
     // knex syntax for selecting things. Look up the documentation for knex for further info
-    const titles = await knex("meals").select("title");
+    const titles = await knex('meals').select('title');
     response.json(titles);
   } catch (error) {
     throw error;
   }
 });
-
 module.exports = router;
+
+//Returns all meals
+mealsRouter.get('/', async (req, res) => {
+  try {
+    const meals = await knex.select('*').from('meals');
+    res.json(meals);
+  } catch (error) {
+    console.log(error);
+    res.status(500).json({error:'error occurred!'});
```

Comment:
The status code 500 already implies that an internal server error has occurred. So no need to add the 'error occurred' part.